nano/node/websocket.cpp

```diff
@@ -4,6 +4,54 @@
 #include <nano/node/node.hpp>
 #include <nano/node/websocket.hpp>
 
+nano::websocket::confirmation_options::confirmation_options (boost::property_tree::ptree const & options_a, nano::node & node_a) :
+all_local_accounts (options_a.get<bool> ("all_local_accounts", false)),
+node (node_a)
+{
+	auto accounts_l (options_a.get_child_optional ("accounts"));
+	if (accounts_l)
+	{
+		for (auto account_l : *accounts_l)
+		{
+			// Check if the account is valid, but no error handling if it's not, simply not added to the filter
+			nano::account result_l (0);
+			if (!result_l.decode_account (account_l.second.data ()))
+			{
+				accounts.insert (account_l.second.data ());
```

Comment:
Probably it can be unordered_set of nano:account or insert result_l.to_account () to prevent nano_ / xrb_ prefixes issues (i.e. you can request array of xrb_accounts or even xrb-accounts, but node message will contain nano_accounts by default)

Reply:
Encoding back is a great idea, thanks.

Reply:
Merged master, `xrb_` and `nano_` both working.

Reply:
Good catch and good fix 👍